docs/src/index.md

```diff
@@ -400,6 +400,71 @@ julia> @rtransform df @passmissing :x = parse(Int, :x_str)
    3 │ missing  missing
 ```
 
+## Passing keyword arguments to underlying DataFrames.jl functions
+
+All DataFramesMeta.jl macros allow passing of keyword arguments to their DataFrames.jl
```

Comment:
can you add a list of macros for which this is supported (e.g. does `@orderby` support it?)